src/ai/susi/server/api/cms/ListSkillService.java

```diff
@@ -358,19 +365,22 @@ else if (filter_type.equals("feedback")) {
             }
         }
 
-        skillStats.put("totalSkills", totalSkills);
-        skillStats.put("reviewedSkills", reviewedSkills);
-        skillStats.put("nonReviewedSkills", nonReviewedSkills);
-        skillStats.put("editableSkills", editableSkills);
-        skillStats.put("nonEditableSkills", nonEditableSkills);
-        skillStats.put("staffPicks", staffPicks);
-        skillStats.put("systemSkills", systemSkills);
+        if(shouldReturnSkillStats) {
+            JSONObject skillStats = new JSONObject();
+            skillStats.put("totalSkills", totalSkills);
```

Comment:
Should this not be in snake case?

Reply:
Yes @arundhati24 , it should ideally be. But, this erroneous naming is used in the entire file and would break the key value used in the different client repositories. Also, I plan to correct the naming in a future PR that would handle all the naming related issues. It seems to be a bit out of scope of the issue.

Reply:
Okay @akshatnitd 👍